packages/xsnap/lib/console-shim.js

```diff
@@ -52,4 +49,12 @@ const console = {
   timeStamp: noop,
 };
 
+let quoteSet = false;
+
+export function setQuote(f) {
+  if (quoteSet) throw TypeError('quote already set');
```

Comment:
Jessie does not include braceless control flow.
```suggestion
  if (quoteSet) {
    throw TypeError('quote already set');
  }
```